app/views/organisations/_organisation.html.slim

```diff
@@ -1,127 +1,127 @@
+        - else
+          - row.with_key text: t("organisations.show.vacancy.job_role"), classes: "govuk-body-s govuk-!-font-weight-bold govuk-!-padding-bottom-0"
+          - row.with_value text: humanize_array(vacancy.job_roles), classes: "govuk-body-s govuk-!-padding-bottom-0"
 
-          - if vacancy.salary?
-            = render "organisations/vacancy_field", summary_list: summary_list, field_desc: t("organisations.show.vacancy.full_time_equivalent_salary"), field_value: vacancy.salary
+      - if vacancy.salary?
+        = render "organisations/vacancy_field", summary_list: summary_list, field_desc: t("organisations.show.vacancy.full_time_equivalent_salary"), field_value: vacancy.salary
 
-          - if vacancy.actual_salary?
-            = render "organisations/vacancy_field", summary_list: summary_list, field_desc: t("organisations.show.vacancy.actual_salary"), field_value: vacancy.actual_salary
+      - if vacancy.actual_salary?
+        = render "organisations/vacancy_field", summary_list: summary_list, field_desc: t("organisations.show.vacancy.actual_salary"), field_value: vacancy.actual_salary
 
-          - if vacancy.pay_scale?
-            = render "organisations/vacancy_field", summary_list: summary_list, field_desc: t("jobs.pay_scale"), field_value: vacancy.pay_scale
+      - if vacancy.pay_scale?
+        = render "organisations/vacancy_field", summary_list: summary_list, field_desc: t("jobs.pay_scale"), field_value: vacancy.pay_scale
 
-          - if vacancy.hourly_rate?
-            = render "organisations/vacancy_field", summary_list: summary_list, field_desc: t("jobs.hourly_rate"), field_value: vacancy.hourly_rate
+      - if vacancy.hourly_rate?
+        = render "organisations/vacancy_field", summary_list: summary_list, field_desc: t("jobs.hourly_rate"), field_value: vacancy.hourly_rate
+
+      - summary_list.with_row do |row|
+        - row.with_key text: t("organisations.show.vacancy.working_pattern"), classes: "govuk-body-s govuk-!-font-weight-bold govuk-!-padding-bottom-0"
+        - row.with_value text: vacancy_working_patterns(vacancy), classes: "govuk-body-s govuk-!-padding-bottom-0 govuk-list"
 
-          - summary_list.with_row do |row|
-            - row.with_key text: t("organisations.show.vacancy.working_pattern"), classes: "govuk-body-s govuk-!-font-weight-bold govuk-!-padding-bottom-0"
-            - row.with_value text: vacancy_working_patterns(vacancy), classes: "govuk-body-s govuk-!-padding-bottom-0 govuk-list"
+    hr.govuk-section-break.govuk-section-break--m.govuk-section-break--visible
 
-        hr.govuk-section-break.govuk-section-break--m.govuk-section-break--visible
+- if organisation.school? && organisation.part_of_a_trust? && organisation.trust.all_vacancies.count != vacancies.count
+  p = govuk_link_to t(".see_trust_jobs", name: organisation.trust.name, count: organisation.trust.all_vacancies.count), organisation_path(organisation.trust)
 
```

Comment:
I think the PR is still missing the last point on the ticket regarding adding a message on top if the School doesn't have vacancies: 

![image](https://github.com/user-attachments/assets/b728fac1-e1f7-4bae-91df-1d0af7433f63)